src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,252 @@
+formatted in BIDS, not the input camera footage.
+
+The extension is not limited to motion data in physical space but also encompasses
+simulated movement in virtual space, as far as these are comparable to movements
+in physical space. The extension is also not limited to the positions/orientations
+of human body parts, other dynamic objects (physical or virtual) in the environment
+whose motion is tracked may be included as additional indications of motion.
+
+For the current BEP, motion data MUST be stored in a `*_motion.tsv` file. A tracking system is defined as a group of tracking devices that share hardware properties (same recording device) and/or software properties (for example same sampling rates). For example if the position of multiple optical markers is processed via one recording unit, this would be defined as a tracking system. Note that it is not uncommon to have multiple tracking systems to record at the same time. Each tracking system should have its own `*[tracksys-<label>]_motion.tsv` file. One column per channel per tracking system is intended in the `*[tracksys-<label>]_motion.tsv` file. The header of each column should correspond to one entry in a `*_channels.tsv` file. All relevant metadata about a tracking systems is stored in the sidecar `*_motion.json` file in the subfield trackingsystem. The data from each tracking system in their original format, if different from `.tsv`, can be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data). The original data format might additionally hold more metadata than currently being specified in the `*_motion.json` file.
+
+### Sidecar JSON (`*_motion.json`)
+
+Generic fields (shared with other BIDS modalities) MUST be present:
+
+| **Key name** | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                   |
+| ------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TaskName     | REQUIRED              | string        | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this TaskName field by removing all non-alphanumeric `[a-zA-Z0-9]` characters. |
+
+Generic fields (shared with other BIDS modalities) SHOULD be present: For consistency between studies and institutions, we
+encourage users to extract the values of these fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+| **Key name**                | **Requirement level** | **Data type** | **Description**                                                                                     |
```

Comment:
such tables AFAIK now should be produced using `MACROS___make_metadata_table` from the `src/schema` -- check other examples in the specification

Reply:
We agreed on using simple MD at first and update to MACROS when everything passed